be/src/exec/partition/partition_hash_map.h

```diff
@@ -390,12 +413,12 @@ struct PartitionHashMapWithOneStringKey : public PartitionHashMapBase<false, fal
 
     PartitionHashMapWithOneStringKey(int32_t chunk_size) : PartitionHashMapBase(chunk_size) {}
 
-    template <typename NewPartitionCallback, typename PartitionChunkConsumer>
+    template <bool EnableDowngrade, typename NewPartitionCallback, typename PartitionChunkConsumer>
```

Comment:
suggest to change EnableDowngrade and is_downgrade to EnablePassthrough, is_passthrough,  downgrade/upgrade is already used in conversion between BinaryColumn and LargeBinaryColumn, it seems quite confusing.

Reply:
Ok, sounds better, I'll update.